client/components/sideCarousel.jsx

```diff
@@ -19,18 +20,41 @@ const Image = styled.img`
   height: 60px;
   width: 60px;
   border-radius: 6px;
+  opacity: 0.6;
+`;
+
+const ClickedListItem = styled.li`
+height: 56px;
+width: 56px;
+margin: 0px 6px 6px 0px;
+border: 2px solid #000;
+border-radius: 6px;
+`;
+
+const ClickedImage = styled.img`
+  height:56px;
+  width:56px;
+  border-radius: 6px;
+  opacity: 1;
 `;
 
 function SideCarousel(props) {
-  const { pictures } = props;
+  const { pictures, currPicPos } = props;
```

Comment:
This is my opinion but I think actually spelling out your variable names leads to easier to read code and with autocomplete in VSCode it's really not much more typing. So I would call this currentPicturePosition so there is no doubt what you mean. It is also extra helpful when working with engineers whose first language isn't English.